source/agora/consensus/EnrollmentManager.d

```diff
@@ -197,25 +197,30 @@ public class EnrollmentManager
 
     /***************************************************************************
 
-        Update the enrolled height of the validatorSet DB.
+        Add a validator to the validator set or update the enrolled height.
 
         Params:
             enroll_hash = enrollment blockheight to update enroll hash
```

Comment:
`enroll_hash = enrollment blockheight to update enroll hash`
 It has been changed to `Enrollment enroll`
It has not been changed but became Merge.

Reply:
https://github.com/bpfkorea/agora/pull/781